apps/shooting-clubs/controllers/confirm.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../common/controllers/confirm');
+
+module.exports = class ConfirmController extends Controller {
+
+  getFieldData(field, req) {
+    const defaults = super.getFieldData(field, req);
+
+    if (field.field === 'location-addresses') {
+      return defaults.value.map(address => {
+        address['address-category'] = [].concat(address['address-category']);
```

Comment:
Sure, we could. I just don't see why it's better.

And with hindsight this normalisation of values should probably get done at the field format level. It's nasty that a field can either yield a string or an array depending on user input, and it should be fixed at source rather than where it has effects.